tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>Not sure we should mention that the bottom detuning might not be defined</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes I agree it makes things a bit too complex here.</p>